corehq/util/metrics/datadog.py

```diff
@@ -61,6 +61,13 @@ def _gauge(self, name: str, value: float, tags: Dict[str, str] = None, documenta
         dd_tags = _format_tags(tags)
         _datadog_record(statsd.gauge, name, value, dd_tags)
 
+    def _datadog_histogram(self, metric_name, metric_value):
+        """
+        https://docs.datadoghq.com/metrics/types/?tab=histogram#metric-types
+        """
+        print(f"Logging {metric_name} - {metric_value}")
+        _datadog_record(statsd.histogram, metric_name, metric_value)
```

Comment:
FYI, histogram stats in datadog are only useful in very specific scenarios because the aggregation window is a constant defined on the client, which means the data will not be useful unless you have a meaningful sample of events consistently recorded within that time period. Unfortunately most of the links I used in the past are now broken, but some discussion on this topic can be found [here](https://github.com/dimagi/commcare-hq/pull/17030#discussion_r127495077) and [here](https://github.com/dimagi/commcare-hq/pull/17080#issuecomment-316395782).

Reply:
We have implemented a different histogram recording technique, which uses counter buckets. You can read more about it here:

https://github.com/dimagi/commcare-hq/blob/4d7b39cc6f8438874a66745136aa67ef75ea9da6/corehq/util/metrics/datadog.py#L64-L90

Reply:
Thank you, I'll read through these references. Will try to use existing 'histogram' metric instead of adding datadog's native histogram.